src/item.cpp

```diff
@@ -28,7 +28,10 @@
 static const std::string GUN_MODE_VAR_NAME( "item::mode" );
 static const std::string CHARGER_GUN_FLAG_NAME( "CHARGE" );
 static const std::string CHARGER_GUN_AMMO_ID( "charge_shot" );
-
+bool     furred = 0;
+bool     pocketed = 0;
+bool     leather_padded = 0;  //Move these somewhere reasonable
+bool     kevlar_padded = 0;
```

Comment:
These are global variables, they are not item specific, they are shared between *all* item instances. You have to add them to the header file as class members. However I suggest you use the `item::item_tags` set instead, or if you need more than a simple boolean use the `item::item_vars` and access it through `item::get_var` etc. (see documentation there).

```C++
if(item_tags.count("pockets") > 0) {
    // => has pockets
} else {
    item_tags.insert("pockets"); // adds pockets
}
```

Using `item::item_tags` has several advantages:
- No space is wasted. Most item instance won't ever need that member, so why add it to all of them?
- It prevents stacking of items that only differ in those modifications - currently modified items would stack with unmodified items, items with different `item_tags` content do not stack.
- It is automatically saved and loaded, no need for you to do it.

Style issue (=personal opinion): booleans should be initialized with another boolean, not an int: `bool x = false;`